solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -15,12 +24,17 @@
 <script>
 
 import exportFromJSON from 'export-from-json'
+import SimpleSpinner from "../../../../../regulations/js/src/components/SimpleSpinner.vue";
```

Comment:
We don't have to dig up and out to another directory like this -- the vite config file has aliases to make this easier.  See here:

https://github.com/CMSgov/cmcs-eregulations/blob/main/solution/ui/regulations/eregs-vite/vite.config.js

So to import SimpleSpinner, all you would need to do is:

`import SimpleSpinner from "legacy/js/src/components/SimpleSpinner.vue";`